packages/grafana-ui/src/components/FormGroup/FormGroup.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { SFC } from 'react';
+import { Label } from '..';
+
+export interface Props {
+  label: string;
+  inputProps: {};
```

Comment:
What do you think about `Props` interface extending `React.InputHTMLAttributes` instead of using `inputProps`?

Reply:
```typescript
export interface Props extends React.InputHTMLAttributes<HTMLInputlement> {
  label: string
  ...
}
```

then, you have all input attributes available without the need for defining them (as you have started to do with`inputProps: {}`).

Later on you would unpack only those attributes that you are interested in, as the rest of them are comming from `React.InputHTMLAttributes`. 

So this:
`const FormGroup: SFC<Props> = ({ label, labelWidth, inputProps, inputWidth })`
will turn into  this:
`const FormGroup: SFC<Props> = ({ label, labelWidth, inputWidth, ...inputProps })`

Reply:
Cool!